.eslintrc.js

```diff
@@ -3,16 +3,25 @@ module.exports = {
   extends: [
     'plugin:@newrelic/eslint-plugin-newrelic/react',
     'plugin:@newrelic/eslint-plugin-newrelic/prettier',
+    'plugin:@typescript-eslint/recommended',
     'plugin:jsx-a11y/recommended',
     'plugin:react-hooks/recommended',
   ],
   // https://github.com/yannickcr/eslint-plugin-react#configuration
-  plugins: ['react', 'jsx-a11y'],
+  plugins: ['react', 'jsx-a11y', '@typescript-eslint'],
   settings: {
     react: {
       version: 'detect',
     },
   },
+  overrides: [
+    {
+      files: ['**/*.{ts,tsx}'],
+      parserOptions: {
+        project: ['./tsconfig.json'],
+      },
+    },
+  ],
```

Comment:
Enables typescript linter for javascript (to help with integrating throughout time) and typescript files